examples/live/check_results.py

```diff
@@ -6,203 +6,230 @@
-            spin1z=sngl_inspiral_table.get_column('spin1z')[0] 
-            spin2z=sngl_inspiral_table.get_column('spin2z')[0] 
-            log.info('IFO SNRs: '+str(snr_list))
-            log.info('Network SNR: '+str(network_snr))
+        sngl_inspiral_table = lsctables.SnglInspiralTable.get_table(xmldoc)
+        
+        trig_props['tc'][x]=sngl_inspiral_table.get_column('end_time')[0]                   
+        trig_props['mass1'][x]=sngl_inspiral_table.get_column('mass1')[0]
+        trig_props['mass2'][x]=sngl_inspiral_table.get_column('mass2')[0]    
+        trig_props['spin1z'][x]=sngl_inspiral_table.get_column('spin1z')[0] 
+        trig_props['spin2z'][x]=sngl_inspiral_table.get_column('spin2z')[0] 
+        
+        
+        snr_list=sngl_inspiral_table.get_column('snr')
+        network_snr = sum([snr ** 2 for snr in snr_list]) ** 0.5      
+        trig_props['net_snr'][x]=network_snr
             
-            if print_template_parameters:
-                  log.info('IFO End Times: '+str(end_time)) 
-                  log.info('Mass 1: %f', mass1)
-                  log.info('Mass 2: %f', mass2)
-                  log.info('Spin1z: %f', spin1z)
-                  log.info('Spin2z: %f', spin2z)
-            
-            #check if trigger properties match injection's properties
-            for t in end_time:
-                  if not close(t,inj_time,1.0):    
-                        coinc_fail = True
-                        log.error('Trigger time does not match injection time')  
-            if not close(mass1, inj_mass1, 5e-7):
-                  coinc_fail = True
-                  log.error('Trigger mass 1 does not match injection mass 1')
-            if not close(mass2, inj_mass2, 5e-7):
-                  coinc_fail = True                  
-                  log.error('Trigger mass 2 does not match injection mass 2')
-            if not close(spin1z, inj_spin1z, 5e-7):
-                  coinc_fail = True
-                  log.error('Trigger spin1z does not match injection spin1z')
-            if not close(spin2z, inj_spin2z, 5e-7):
-                  coinc_fail = True
-                  log.error('Trigger spin2z does not match injection spin1z')   
-            if not close(network_snr, inj_snr, 0.1):
-                  coinc_fail = True
-                  log.error('Network SNR test failed')
+        
+        log.info('IFO SNRs: '+str(snr_list))
+        log.info('Network SNR: %f', network_snr)
+        log.info('IFO End Time: %f', trig_props['tc'][x]) 
+        log.info('Mass 1: %f', trig_props['mass1'][x])
+        log.info('Mass 2: %f', trig_props['mass2'][x])
+        log.info('Spin1z: %f', trig_props['spin1z'][x])
+        log.info('Spin2z: %f', trig_props['spin2z'][x])
+     
+
+    # check if injections match trigger params
+    for i in range(n_injs):
+        has_match = False
+        for j in range(n_coincs):
+            if close(inj_time[i],trig_props['tc'][j],1.0):
```

Comment:
I think the match logic got a bit overcomplicated and could be simplified as follows. Start with `has_match = False` as you did. Then for each coinc, check if the coinc matches *all* relevant parameters (tc, mass1, mass2, spin1z, spin2z, network SNR). If everything matches, set `has_match` to true and break out of the `j` loop. Otherwise keep looping.

However, this will not work if your want to have a log message that says exactly which parameter did *not* match. I am not sure this is necessary for the Travis test, because if the test fails we would probably run a more verbose local check anyway. But if you absolutely want this log message, you can keep the code as is.